### PR TITLE
vagov-1437: adding wrapping divs

### PR DESCRIPTION
**Pull request by mojaray2k**
## Description
*As a screen reader user, I want to hear the alertbox read out when I enter an invalid routing number".

## Testing done
I do not know how to trigger this error. 

## Screenshots
Fullscreen_9_10_19__2_56_PM

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/personalization/profile360/components/PaymentInformationEditModalError.jsx
Comment (ncksllvn) [inline src/applications/personalization/profile360/components/PaymentInformationEditModalError.jsx]: I think you can just add the `role` and `aria-atomic` attributes to the `AlertBox` component at the bottom, https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/personalization/profile360/components/PaymentInformationEditModalError.jsx#L64.

Triggering this error is tricky I think because of limitations with our local API setups, but this error shows when the lookup for a bank name from a routing number fails.

**Comment by erikphansen** [comment]
@mojaray2k It looks like we just need to wrap the `<PaymentInformationEditModalError>` in the `div` Trevor asks for:

```
<!-- This empty div should be in the modal on first render -->
<div id="errors" role="alert" aria-atomic="true">
  <!-- Now add the alert box when no valid routing number is found -->
  <div class="usa-alert usa-alert-error">
    <div class="usa-alert-body">
      <div class="usa-alert-text">
        <p>We couldn’t find a bank linked to this routing number. Please check your bank’s 9-digit routing number and enter it again.</p>
      </div>
    </div>
  </div>
</div>
```

**Comment by ncksllvn** [comment]
Lol @erikphansen is correct on that! My suggestion didn't make sense for a React component.

**Comment by rianfowler** [comment]
Please tag the review group when this has an approval from a team member. We recommend using the [draft pull request](https://github.blog/2019-02-14-introducing-draft-pull-requests/) until this has an approval from a team member. You can see the review guidelines for this repo [here](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/code_review_guidelines.md)

Thanks!

**Comment by mojaray2k** [comment]
@erikphansen and @ncksllvn That change has been updated.

File: src/applications/personalization/profile360/components/PaymentInformationEditModal.jsx
Comment (erikphansen) [inline src/applications/personalization/profile360/components/PaymentInformationEditModal.jsx]: The `PaymentInformationEditModalError` component renders out the DOM starting with this `div` right here, so you don't need to add it to the markup. We just need to wrap the existing `PaymentInformationEditModalError` component in the `div` you added at line 121.

**Review by erikphansen — CHANGES_REQUESTED** [review]

File: src/applications/personalization/profile360/components/PaymentInformationEditModal.jsx
Comment (mojaray2k) [reply src/applications/personalization/profile360/components/PaymentInformationEditModal.jsx]: Aren't I doing that? Or maybe I am missing something?

Comment (erikphansen) [reply src/applications/personalization/profile360/components/PaymentInformationEditModal.jsx]: No you're wrapped the `PaymentInformationEditModalError` in a bunch of extra markup that isn't needed because that extra markup is exactly what the `PaymentInformationEditModalError` component renders. That component simply renders out an `AlertBox` component and you can find the source code of the `AlertBox` component here: https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/47dcd54d0ac7cc8917452a4328d108630cd79db7/packages/formation-react/src/components/AlertBox/AlertBox.jsx. So you're effectively wrapping an AlertBox in another AlertBox.

And actually reviewing Trevor's original suggestion for the fix, the wrapper div, `<div id="errors" role="alert" aria-atomic="true">` should live _outside_ of the conditional logic that renders the `PaymentInformationEditModalError` component if `props.responseError` is set.

Comment (mojaray2k) [reply src/applications/personalization/profile360/components/PaymentInformationEditModal.jsx]: Oh ok. Dang sorry about the long way round explanation.

Comment (mojaray2k) [reply src/applications/personalization/profile360/components/PaymentInformationEditModal.jsx]: Ok I just push that code.

File: src/applications/personalization/profile360/components/PaymentInformationEditModal.jsx
Comment (erikphansen) [inline src/applications/personalization/profile360/components/PaymentInformationEditModal.jsx]: Trevor said that this `div` should always be on the page and the Alert should be conditionally added within it. So this `dvi` should wrap the conditional logic started on line 120

Comment (mojaray2k) [reply src/applications/personalization/profile360/components/PaymentInformationEditModal.jsx]: Done

**Review by erikphansen — APPROVED** [review]

**Comment by erikphansen** [comment]
All yours, @department-of-veterans-affairs/frontend-review-group !

**Review by rianfowler — APPROVED** [review]

**Pull request merged**
